src/main/java/prefs/GlobalConfig.java

```diff
@@ -82,6 +84,14 @@ public void addBoard(String name, List<PanelInfo> panels) {
     public void removeBoard(String name) {
         savedBoards.remove(name);
     }
+    
+    public void setLastOpenBoard(String board) {
+        lastOpenBoard = Optional.of(board);
+    }
+    
+    public String getLastOpenBoard() {
+        return lastOpenBoard.get();
```

Comment:
The point of `Optional` is to express statically that a value may be absent, and force consumers of the value to check for this (unlike `null`, which doesn't make you check anything).

Calling `get` without checking `isPresent` will fail at runtime (similar to dereferencing a null pointer). You should instead propagate the `Optional` out to callers of this method.